docs/develop/go/core-application.mdx

```diff
@@ -86,161 +86,19 @@ tags:
 description: The Foundations section of the Temporal Developer's guide introduces essential concepts for building and running a Temporal Application, outlining steps to start Workflow and Activity Execution, running development Workers, and installing the Temporal CLI.
 ---
 
-The Foundations section of the Temporal Developer's guide covers the minimum set of concepts and implementation details needed to build and run a [Temporal Application](/temporal#temporal-application)—that is, all the relevant steps to start a [Workflow Execution](#develop-workflows) that executes an [Activity](#activity-definition).
+This page shows how to do the following:
 
 In this section you can find the following:
 
-- [Run a development Temporal Service](#run-a-development-server)
 - [Develop a Workflow](#develop-workflows)
 - [Develop an Activity](#activity-definition)
 - [Start an Activity Execution](#activity-execution)
 - [Run a dev Worker](#develop-worker)
 - [Run a Temporal Cloud Worker](#run-a-temporal-cloud-worker)
 
-## How to install the Temporal CLI and run a development server {#run-a-development-server}
+## Develop a basic Workflow {#develop-workflows}
 
-This section describes how to install the [Temporal CLI](/cli) and run a development Temporal Service.
```

Comment:
This is HUGE removal of material. Just making sure it's intentional.

Reply:
Haven't looked back at this but any removals are likely because they are being moved to a new spot to align with the migration. The .NET one was a manual migration since the rest weren't, need to make sure they are still manually moved. ✔️